### ISSUE TITLE
Subtitles error: "Expected WEBVTT. Got null"

### ISSUE THREAD

**Issue by ihauh** [yuliskov/SmartTube#3733]
Since yesterday I get this error when trying to display subtitles (either generated or proper subs): 

> Selected SUBTITLE format isn't supported.
> Try to select another one by long pressing CC button in the player.
> If this won't help try to reboot the device.
> com.google.android.exoplayer2.ParserException: Expected WEBVTT. Got null

And the video won't play. Selecting a different one or rebooting doesn't help, I can only disable them and it will play the video. Subs on the official YT app do work.
This is a Xiaomi Android TV.

Thanks!

**Comment by BuzzFuzz** [comment]
Yup, same here. I need to disable every single channel sub to be able to watch it.

**Comment by ihauh** [comment]
Version 22.60 seems to have fixed this bug.

**Comment by lucasenicora** [comment]
Same here. I can still reproduce the issue on v22.81

**Comment by ihauh** [comment]
I don't update anymore unless the app is completely broken. Still working for me, v 22.70.
